extensions/admin-tool-ext-win/src/test/extension.test.ts

```diff
@@ -1,13 +0,0 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the Source EULA. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-import 'mocha';
-import * as vscode from 'vscode';
-
-describe('Extension activate test', () => {
-	it('Extension should activate correctly', async function (): Promise<void> {
```

Comment:
Why was this deleted?

Reply:
oops, that was a change that wasn't meant to be pushed.  thanks!